data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -949,27 +870,7 @@
 query = tpl.render(new_rdr=new_rdr, project_id=project_id)
 execute(client, query)
 
-# # Checks for basics survey module
-# Participants with data in other survey modules must also have data from the basics survey module.
-# This check identifies survey responses associated with participants that do not have any responses
-# associated with the basics survey module.
-# In ideal circumstances, this query will not return any results.
-
-tpl = JINJA_ENV.from_string('''
-SELECT DISTINCT person_id FROM `{{project_id}}.{{new_rdr}}.observation`
-JOIN `{{project_id}}.{{new_rdr}}.concept` on (observation_source_concept_id=concept_id)
-WHERE vocabulary_id = 'PPI' AND person_id NOT IN (
-SELECT DISTINCT person_id FROM `{{project_id}}.{{new_rdr}}.concept`
-JOIN `{{project_id}}.{{new_rdr}}.concept_ancestor` on (concept_id=ancestor_concept_id)
-JOIN `{{project_id}}.{{new_rdr}}.observation` on (descendant_concept_id=observation_concept_id)
-WHERE concept_class_id='Module'
-AND concept_name IN ('The Basics')
-AND questionnaire_response_id IS NOT NULL)
-''')
-query = tpl.render(new_rdr=new_rdr, project_id=project_id)
-execute(client, query)
```

Comment:
Removing basics queries from the raw notebook. These queries should be checked in the clean notebook instead and already exist there.